nixops/resources/aws_vpn_connection_route.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+
+import botocore
+
+import nixops.util
+import nixops.resources
+from nixops.resources.ec2_common import EC2CommonState
+import nixops.ec2_utils
+from nixops.diff import Diff, Handler
+from nixops.state import StateDict
+
+class AWSVPNConnectionRouteDefinition(nixops.resources.ResourceDefinition):
+    """Definition of a VPN connection route"""
+
+    @classmethod
+    def get_type(cls):
+        return "aws-vpn-connection-route"
+
+    @classmethod
+    def get_resource_type(cls):
+        return "awsVPNConnectionRoutes"
+
+    def show_type(self):
+        return "{0}".format(self.get_type())
```

Comment:
I wonder if we can eliminate having a separate `<Resource>Definition` class. It's pretty much all boilerplate code at this point...